docs/EARLY-RELEASE.md

```diff
@@ -28,9 +29,10 @@ big and we never release just a patch. There is only "release".
  - Is there a security advisory rated high or critical?
  - Is there a data corruption bug?
  - Did the bug cause an API/ABI breakage?
+ - Will the problem annoy an non-insignificant share of the user population?
 
-If the answer is yes to one or more of the above, an early release might
-indeed be warranted.
+If the answer is yes to one or more of the above, an early release might be
+warranted.
 
 More questions to ask ourselves when doing the assessment if the answers to
 the three ones above are all 'no'.
```

Comment:
(this comment is for line 52 but GitHub doesn't let me leave it there). Change "How long is it left" to "How long is it" (or "How long is left").

(line 60) If we're lowering the bar for patch releases. "Unless done for security or similarly important reasons" seems to strongly worded. And "two weeks" should be now "10 days".